pinctrl/hw_pwm.go

```diff
@@ -11,9 +11,8 @@ import (
 
 	mmap "github.com/edsrzf/mmap-go"
 	"github.com/pkg/errors"
```

Comment:
It snuck in through here, if we're only using `New` it should be a simple find and replace, if we're using errors.Errorf or Wrap we should replace those with fmt.Errorf and errros.Join respectively.